metadata.rb

```diff
@@ -10,15 +10,11 @@
 
 supports 'ubuntu'
```

Comment:
make sure to set chef_version here to >= 15.0 now that you're using archive_file